Vagrantfile

```diff
@@ -106,6 +106,7 @@ Vagrant.configure("2") do |config|
       # Only execute once the Ansible provisioner,
       # when all the machines are up and ready.
       if i == $num_instances
+        config.vm.provision :shell, inline: '[[ -x "$(command -v yum)" ]] && ifup eth1', privileged: true
```

Comment:
I have just built image for centos 7.2 with several interfaces. Vagrant brought them up as expected.